### ISSUE TITLE
Incorrect width and height in convolutional layer.

### ISSUE THREAD

**Issue by diegogcerdas** [maxvanspengler/hyperbolic_learning_library#33]
Hello, I am trying to use an `HConvolution2d` layer, but I am getting an error regarding the output shape.

**Code**:
```
manifold = PoincareBall(Curvature(1))
layer = hnn.HConvolution2d(3, 64, kernel_size=11, stride=4, padding=2, manifold=manifold)

x = torch.randn(1, 3, 224, 224)
tangents = TangentTensor(data=x, man_dim=1, manifold=manifold)
x_t = manifold.expmap(tangents)
out = layer(x_t)
```

**Raises**:
```
RuntimeError: shape '[1, 64, 54, 54]' is invalid for input of size 193600
```

Perhaps a better way to compute `out_height` and `out_width` in `hypll/nn/modules/convolution.py` [lines 88-89](https://github.com/maxvanspengler/hyperbolic_learning_library/blob/main/hypll/nn/modules/convolution.py#L88) is the following:

```
out_height = int(np.floor((height - self.kernel_size[0] + 2 * self.padding) / self.stride)) + 1
out_width = int(np.floor((width - self.kernel_size[1] + 2 * self.padding) / self.stride)) + 1
```
which returns an output shape `[1, 64, 55, 55]`.

**Comment by philippmwirth** [comment]
Hi @diegogcerdas thanks a lot for reporting this issue! I'll take a look asap and will let you know about any updates.

**Comment by philippmwirth** [comment]
Fix was merged in #34. To try it out, install from source with 
```
git clone https://github.com/maxvanspengler/hyperbolic_learning_library.git
cd hypll
pip install .
```

Thanks again for reporting and please let us know if there are any other problems!
